src/include/cell.i

```diff
@@ -756,15 +782,19 @@ restart:
             break;
         flags = *p++; /* skip second descriptor byte */
 
-        if (LF_ISSET(WT_CELL_TS_DURABLE))
-            WT_RET(__wt_vunpack_uint(
-              &p, end == NULL ? 0 : WT_PTRDIFF(end, p), &unpack->newest_durable_ts));
+        if (LF_ISSET(WT_CELL_PREPARE))
+            F_SET(unpack, WT_CELL_UNPACK_PREPARE);
         if (LF_ISSET(WT_CELL_TS_START))
             WT_RET(__wt_vunpack_uint(
               &p, end == NULL ? 0 : WT_PTRDIFF(end, p), &unpack->oldest_start_ts));
         if (LF_ISSET(WT_CELL_TXN_START))
             WT_RET(__wt_vunpack_uint(
               &p, end == NULL ? 0 : WT_PTRDIFF(end, p), &unpack->oldest_start_txn));
+        if (LF_ISSET(WT_CELL_TS_DURABLE_START)) {
+            WT_RET(__wt_vunpack_uint(
```

Comment:
Am I understanding correctly that this flag should never be set since this is `v4.2`? I don't really understand the purpose of this PR.

Reply:
That's exactly right.  We should be able to read 4.4 files and not hiccup.  We don't have to do anything with the new timestamp.  What we should do, when we see the the prepare bit, is issue a warning message.  I haven't made that change yet.

Reply:
@kommiharibabu @ddanderson Sure. But I feel like the only thing we need to do is to unpack an extra int from the cell if that flag is set and throw that int value away.

Reply:
@tetsuo-cpp That's true.  One reason to keep it is to help in debugging situations - it may be helpful to see that certain fields/timestamps were set in the file.  The functions in `bt_debug.c` will dump the durable timestamps, etc.  Another possible reason to fully unpack is to keep code lined up between the two versions.  Makes it a little easier to have a handle on correctness, especially for pieces of the code that are only executed in odd cases (like downgrades).

Reply:
> @tetsuo-cpp That's true. One reason to keep it is to help in debugging situations - it may be helpful to see that certain fields/timestamps were set in the file. The functions in `bt_debug.c` will dump the durable timestamps, etc. Another possible reason to fully unpack is to keep code lined up between the two versions. Makes it a little easier to have a handle on correctness, especially for pieces of the code that are only executed in odd cases (like downgrades).

Understood. Thanks @ddanderson and @kommiharibabu.